backend/app/exporters/models/mods.rb

```diff
@@ -158,15 +165,51 @@ def handle_notes(notes)
     end
   end
 
+  # notes relating to extents are treated differently than other notes 
+  # when the model is serialized.
+  def handle_extents_notes(notes)
+    notes.each do |note|
+      next unless note['type'] == 'physdesc' || note['type'] == 'dimensions'
+      next unless note['publish'] == true
+
+      content = ASpaceExport::Utils.extract_note_text(note)
+      mods_note = case note['type']
+                  when 'physdesc'
+                    new_mods_note('note',
```

Comment:
@avatar382 @cdibella From note on line 72 of spreadsheet, shouldn't both the physdesc and dimensions notes be wrapped in a physicalDescription tag? If so, I believe that is the last (5th) input parameter for the new_mods_note method.

Reply:
yes, <physicalDescription> is the wrapper tag for both of these things

Reply:
my tag disappeared, at least on the web page, but yup, physicalDescription is the wrapper